packages/remix-dev/config.ts

```diff
@@ -450,6 +455,15 @@ export async function readConfig(
 
   let serverDependenciesToBundle = appConfig.serverDependenciesToBundle || [];
 
+  // When tsconfigPath is undefined, the default "tsconfig.json" is not
+  // found in the root directory.
+  let tsconfigDefaultFilename = "tsconfig.json"
```

Comment:
do you happen to know if this all works as expected if you use a jsconfig.json file and pass that to esbuild?

Reply:
it appears that it does, can you update this to also check for `jsconfig.json`? i think something like this should be sufficient https://github.com/remix-run/remix/pull/3012/files#diff-852e3f3cc491a6fcc66706c94417be59f4c12569e0f367354613bfc103df7b38R299-R307

Reply:
Sure. There you go!

Reply:
:chefkiss: